src/trans/cpu/external/gpnorm_transad.F90

```diff
@@ -0,0 +1,91 @@
+! (C) Copyright 2008- ECMWF.
```

Comment:
Do we care about accuracy of copyrights here?
If we do, I guess this should be 2024?